src/mic/templates/Dockerfile

```diff
@@ -4,17 +4,29 @@ FROM {{ language.image }}
 ADD requirements.txt /tmp/requirements.txt
 RUN pip3 install -U pip
 RUN pip3 install -r /tmp/requirements.txt
+
 {% endif -%}
 
 {% if "conda" in language.label  -%}
 ADD environment.yml /tmp/environment.yml
 RUN conda env update -f /tmp/environment.yml
+
 {% endif -%}
 
 {% if "generic" in language.label  -%}
 #If you put an executable in the bin directory, uncomment the next file
 #ADD bin/* /usr/bin/
 {% endif -%}
+
 COPY entrypoint.sh /set_umask.sh
+
+{% if os -%}
+{% if os == "windows"-%}
```

Comment:
A user can use a custom image and the image could be not debian.

Reply:
The new fix should address this problem